src/librustc_mir/transform/mod.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+pub mod simplify_cfg;
+mod util;
+
+use repr::Mir;
+
+pub trait MIRPass {
```

Comment:
I just noticed that we actually have [naming conventions](https://aturon.github.io/style/naming.html) written down for that:

> In CamelCase, acronyms count as one word: use Uuid rather than UUID.

(My guess is that this was made to make things like `XMLHTTPRequest` more readable)

In this case, `SimplifyCFG` should probably be `SimplifyCfg` as well ;)